manifests/params.pp

```diff
@@ -8,4 +8,15 @@
     'Ubuntu' => 'rpcbind',
     default  => 'portmap'
   }
+  $portmap_service = $::operatingsystem? {
+    'Debian' => $::lsbdistcodename? {
+      'Wheezy' => 'rpcbind',
+      default  => 'portmap',
+    },
+    'Ubuntu' => $::lsbdistcodename? {
+      'trusty' => 'rpcbind',
+      default  => 'portmap',
+    },
+    default  => 'portmap'
+  }
```

Comment:
If we're going to have two selectors, I'd rather use a case statement instead, and throw in a default case that fails while we're at it.